### PR TITLE
Sort spectra on date

### PR DESCRIPTION
**Pull request by scizen9**
Very simple change to address issue [#81](https://github.com/fritz-marshal/fritz-beta-feedback/issues/81) requesting that spectra be sorted on date

### CODE REVIEW

**Review by dmitryduev — APPROVED** [review]
👍🏻

File: skyportal/plot.py [hunk 1]
```diff
@@ -1032,6 +1032,8 @@ def spectroscopy_plot(
         data.append(df)
     data = pd.concat(data)
 
+    data.sort_values(by=['date_observed', 'wavelength'], inplace=True)
+
```
Comment (guynir42) [inline skyportal/plot.py:1036]: I'm not sure how sorting on wavelength works? I assume each spectrum has a whole array of wavelengths?

Comment (scizen9) [reply skyportal/plot.py:1036]: Yes.  Without including wavelength in the sort, they end up scrambled.

Comment (scizen9) [reply skyportal/plot.py:1036]: I tested having two spectra on the same date, but with different times and it sorted them correctly.  The hover in the spectrum plot only shows date, however, and this should be fixed in a different issue/PR (which I will log).

**Pull request merged**